### ISSUE TITLE
Update all example RDF and image files

### ISSUE THREAD

**Issue by nicholascar** [w3c/dxwg#658]
Dependent on all other prof-2PWD Issues being solved first

**Comment by nicholascar** [comment]
Several PRs update diagrams and examples, principally PR https://github.com/w3c/dxwg/pull/745

**Comment by kcoyle** [comment]
Nick, could you make the changes available here for discussion, please. It's easier to discuss in a github issue than a pull request, and these are substantial changes that should be agreed by the group as a whole. Also, do not close an issue without discussion first. If these changes relate to discussions in progress (and it appears to me that they are) please alert those discussions to the changes for review.

Basically discuss first, do pull request once there is agreement on the changes. Thanks.

**Comment by nicholascar** [comment]
Initial diagram update work completed in early March '19 with PR https://github.com/w3c/dxwg/pull/745. New diagram Issues, such as https://github.com/w3c/dxwg/issues/826 being created for further updates.

**Comment by kcoyle** [comment]
Nick, I asked you to make the changes available for discussion here. Please do that.

**Comment by nicholascar** [comment]
Changes are available for discussion: all changes are seen as PRs and the results of changes - the files within the profilesont/ folder - can all be seen and Issues about them can be raised.

**Comment by nicholascar** [comment]
Closing after a week's listing as "due for closing" in [Plenary 2019-08-27](https://www.w3.org/2017/dxwg/wiki/Meetings:Telecon2019.08.27)

**Comment by aisaac** [comment]
I support the closing: there is nothing to discuss here as the original issue matter sounds more like a reminder assigned to editors than a real issue/comment. One could say that there will always need to be a general check on examples and images, until the document is declared closed for commenting and editing...
